lisa/tools/qemu.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy.assertpy import assert_that
+
+from lisa.executable import Tool
+from lisa.operating_system import Fedora, Posix
+from lisa.tools import Lsmod
+
+
+class Qemu(Tool):
+
+    QEMU_INSTALL_LOCATIONS = ["qemu-system-x86_64", "qemu-kvm", "/usr/libexec/qemu-kvm"]
+    COMMAND = "qemu-system-x86_64"
+
+    @property
+    def command(self) -> str:
+        return self.COMMAND
+
+    def can_install(self) -> bool:
+        return True
+
+    def create_vm(
+        self,
+        port: int,
+        guest_image_path: str,
+    ) -> None:
+        # start vm on the current node
+        # port : port of the host vm mapped to the guest's ssh port
+        # guest_image_path : path of the guest image
+
+        result = self.run(
+            f"-smp 2 -m 2048 -hda {guest_image_path} "
+            "-device e1000,netdev=user.0 "
+            f"-netdev user,id=user.0,hostfwd=tcp::{port}-:22 "
+            "-enable-kvm -display none -daemonize",
+            sudo=True,
+            shell=True,
+        )
+        assert_that(
```

Comment:
It can be in the `run` command. So it doesn't need to be a separated command.

Reply:
I didn't get this comment. Can you please elaborate?

Reply:
the `run` command has two arguments, `expected_exit_code` and `expected_exit_code_failure_message`. Use them, so it doesn't to be written in a separated assert_that.